src/components/layout/account-popup.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import Modal from 'react-modal';
+import styles from './styles.module.css';
```

Comment:
```suggestion
import styles from './layout.module.css';
```
also, please change the respective file name to layout.module.css🙂